### ISSUE TITLE
Error on blank # href

### ISSUE THREAD

**Issue by pickleshb** [wjdp/htmltest#30]
When `CheckInternalHash: false` blank `#` hrefs fail, i.e `href="#"`

**Comment by wjdp** [comment]
Yep, I think we'll keep that behaviour but add another option to toggle the empty hash checking.

Something like `IgnoreEmptyHash`.

Just found the unit test for it: :smile: 

```
func TestAnchorInternalHashOption(t *testing.T) {
	// passes for # href when asked
	t.Skip("Not yet implemented")
	hT := tTestFile("fixtures/links/hash_href.html")
	tExpectIssueCount(t, hT, 0)
}
```
https://github.com/wjdp/htmltest/blob/445c61c/htmltest/check-link_test.go#L312

**Comment by pickleshb** [comment]
Sounds good

**Comment by wjdp** [comment]
Thinking about it though, unsure if this should be included in `CheckInternalHash`

1. It's a separate feature, add option `IgnoreEmptyHash`
2. It's part of `CheckInternalHash` disable with it, also add `IgnoreEmptyHash` to selectively disable it.

**Comment by davidtaylorhq** [comment]
I think option 2 is better

**Comment by pickleshb** [comment]
I thinky option 2 too

**Comment by wjdp** [comment]
Fixed in #31
